modules/social_features/social_group/src/SetGroupsForNodeService.php

```diff
@@ -178,7 +181,7 @@ public static function addGroupContent(NodeInterface $node, Group $group) {
    */
   public static function removeGroupContent(NodeInterface $node, Group $group) {
     // Try to load group content from entity.
-    if ($group_contents = GroupContent::loadByEntity($node)) {
+    if (($group_contents = GroupContent::loadByEntity($node)) && !empty($group_contents)) {
```

Comment:
Ronald, I believe that we could take the same approach here, and move the assign of $group_contents to outside the if, just to be 100% safe.